app/experimenter/nimbus-ui/src/components/FormBranches/FormBranch.tsx

```diff
@@ -85,46 +93,6 @@ export const FormBranch = ({
 
   const handleRemoveClick = () => onRemove && onRemove();
 
-  const formControlCommon = (
```

Comment:
Nice, the swap of `formControlAttrs` for `formControlCommon` looks pretty clean here!